src/EditorFeatures/Core/SymbolSearch/SymbolSearchUpdateEngine.cs

```diff
@@ -211,9 +213,9 @@ private PackageWithTypeResult CreateResult(AddReferenceDatabase database, Symbol
 
             return new PackageWithTypeResult(
                 packageName: packageName,
-                typeName: type.Name.ToString(),
-                version: version,
                 rank: GetRank(type),
+                typeName: type.Name.ToString(),
+                version: string.IsNullOrWhiteSpace(version) ? null : version,
```

Comment:
why the change?

Reply:
Previously this was done in the constructor. The whitespace check however is only needed when the version string comes from nuget. It's not needed in the other callsites of the constructor. It also shouldn't be done when constructing the object by deserialization.